src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/TestRunner.java

```diff
@@ -11,9 +11,10 @@
 import java.util.concurrent.TimeUnit;
 
 public abstract class TestRunner {
+    public static final int defaultTimeout = 20;
 
     @Getter
-    private WebDriver driver;
+    protected static WebDriver driver;
```

Comment:
it's not your class, why did you made changes here?

Reply:
a was pushed master into my branch again. And there was some issues before demo. When i tried to start my tests - Makoviichuk's ElementsUtil class was sending error. Thats why i make it protected and static